emba

```diff
@@ -388,11 +388,22 @@ main() {
   if [[ ${IN_DOCKER} -eq 1 ]] ; then
     # set external path new for docker
     export EXT_DIR="/external"
+    # TODO move to container 2
     export PATH_CVE_SEARCH="${EXT_DIR}""/cve-search/bin/search.py"
   fi
 
   # Check all dependencies of EMBA
-  dependency_check
+  if [[ "${CONTAINER_NUMBER}" -eq 2 ]] ;  then
```

Comment:
could we move this into the dependency check?

Reply:
Possible, but the if-else-tree was getting too big for my liking there, so I put it in here

I can move it if we prefer readability inside the main script

Reply:
I would prefer moving it into the dep checker. Otherwise we are going to mix the dep checker with the main emba script.